pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/patch/ObjectDeleteTest.java

```diff
@@ -0,0 +1,105 @@
+package org.broadinstitute.dsm.patch;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.broadinstitute.dsm.DbAndElasticBaseTest;
+import org.broadinstitute.dsm.db.OncHistoryDetail;
+import org.broadinstitute.dsm.db.Tissue;
+import org.broadinstitute.dsm.db.dao.ddp.instance.DDPInstanceDao;
+import org.broadinstitute.dsm.db.dao.ddp.tissue.TissueDao;
+import org.broadinstitute.dsm.db.dto.ddp.instance.DDPInstanceDto;
+import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantDto;
+import org.broadinstitute.dsm.util.DSMOncHistoryCreatorUtil;
+import org.broadinstitute.dsm.util.ElasticTestUtil;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ObjectDeleteTest extends DbAndElasticBaseTest {
+    private static final DDPInstanceDao ddpInstanceDao = new DDPInstanceDao();
+    private static final String instanceName = "patch_instance";
+    private static final String groupName = "patch_group";
+    static DSMOncHistoryCreatorUtil dsmOncHistoryCreatorUtil;
+    static int participantId;
+    static String userEmail = "patchTestUser1@unittest.dev";
+    static String guid = "PATCH_TEST_PARTICIPANT";
+    static String guid2 = "PATCH_PARTICIPANT";
+    private static String esIndex;
+    private static DDPInstanceDto ddpInstanceDto;
+    private static ParticipantDto testParticipant = null;
+
+    @BeforeClass
+    public static void doFirst() {
+        esIndex = ElasticTestUtil.createIndexWithMappings(instanceName, "elastic/lmsMappings.json");
```

Comment:
It's worth it to take the time to try dump another index here if the main driver is another project like OS.  It's pretty straightforward--ElasticTestUtil has some pointers on how to grab index/settings files from an existing instance.